clang/include/clang/CIR/Dialect/IR/CIRTypes.td

```diff
@@ -253,4 +253,15 @@ def VoidPtr : Type<
       "mlir::cir::VoidType::get($_builder.getContext()))"> {
 }
 
+//===----------------------------------------------------------------------===//
+// Global type constraints
+//===----------------------------------------------------------------------===//
+
+def CIR_StructType : Type<CPred<"$_self.isa<::mlir::cir::StructType>()">, "CIR struct type">;
```

Comment:
Can you break `"CIR struct type"` into the next line?

Reply:
Done.